client/src/elm/Pages/Prenatal/Activity/Utils.elm

```diff
@@ -3593,15 +3593,32 @@ resolveMedicationTreatmentFormInputsAndTasksCommon language currentDate setBoolI
 
                     else
                         ( [], [] )
+
+                stillTakingValue =
+                    if config.stillTakingFormValue == form.hivStillTaking then
+                        div []
+                            [ viewQuestionLabel language Translate.TreatmentReviewQuestionStillTakingForHIV
```

Comment:
This is the only label that should change.
Everything else is code duplication.